Gems/Atom/RPI/Code/Source/RPI.Public/Image/ImageSystem.cpp

```diff
@@ -305,7 +305,7 @@ namespace AZ
 
             auto systemAttachmentImage = RPI::AttachmentImage::Create(createImageRequest);
             m_systemAttachmentImages[format] = systemAttachmentImage;
-            return m_systemAttachmentImages[format];
+            return systemAttachmentImage;
```

Comment:
Those changes trigger `Wreturn-stack-address` on Clang.

Reply:
Verified, the method returns a reference but systemAttachmentImage is a local Data::Instance<AttachmentImage> . Will revert this change, since this is an override method and we don't want to change the signature